source/administration/services/hdbpp/hdbpp-es-interface.rst

```diff
@@ -0,0 +1,272 @@
++------------------------------+-------------------------------------------------------+
+| AttributeNumber              | number of attributes configured for archiving         |
++------------------------------+-------------------------------------------------------+
+| AttributeOkList              | return the list of attributes not in error            |
++------------------------------+-------------------------------------------------------+
+| AttributeOkNumber            | number of archived attributes not in error            |
++------------------------------+-------------------------------------------------------+
+| AttributePausedList          | list of paused attributes                             |
++------------------------------+-------------------------------------------------------+
+| AttributePausedNumber        | number of paused attributes                           |
++------------------------------+-------------------------------------------------------+
+| AttributePendingList         | list of attributes waiting to be archived             |
++------------------------------+-------------------------------------------------------+
+| AttributePendingNumber       | number of attributes waiting to be archived           |
++------------------------------+-------------------------------------------------------+
+| AttributeRecordFreq          | total number of records per time                      |
++------------------------------+-------------------------------------------------------+
+| AttributeRecordFreqList      | per-attribute number of records per time              |
++------------------------------+-------------------------------------------------------+
+| AttributeStartedList         | list of started attributes                            |
++------------------------------+-------------------------------------------------------+
+| AttributeStartedNumber       | number of started attributes                          |
++------------------------------+-------------------------------------------------------+
+| AttributeStoppedList         | list of stopped attributes                            |
++------------------------------+-------------------------------------------------------+
+| AttributeStoppedNumber       | number of stopped attributes                          |
++------------------------------+-------------------------------------------------------+
+| Context                      | archiver current context (r/w)                        |
++------------------------------+-------------------------------------------------------+
+| StatisticsResetTime          | seconds elapsed since last statistics reset           |
++------------------------------+-------------------------------------------------------+
+
+Table 2: Event Subscriber Attributes.
+
+The class and device properties availabile for configuration are shown
+in table. According to TANGO
```

Comment:
..table 3.

Reply:
I am missing something here…
table 1 are the command, and this one should be 2 no ?

Reply:
No. we're talking about properties, so should be table 3. Table 2 is about attributes.

Reply:
Got it, thx.